roles/reportinator_server/tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Set reportinator_server dir
+  ansible.builtin.set_fact:
+    reportinator_server_dir: "{{ homedir }}/services/reportinator_server"
+
+- name: Ensure services/reportinator_server exists
+  ansible.builtin.file:
+    path: "{{ reportinator_server_dir }}"
+    state: directory
+    mode: '0755'
+
+
+- name: Copy necessary template files to reportinator_server dir
+  ansible.builtin.template:
+    src: "{{ item.src }}"
+    dest: "{{ reportinator_server_dir }}/{{ item.dest }}"
+    mode: 0644
+  loop:
```

Comment:
If you like you can omit the `loop` and just set the `src` and `dest` above. It's more readable but this is more convenient if we add more template files in the future.